src/components/Dropdown.jsx

```diff
@@ -31,4 +32,16 @@ class Dropdown extends React.Component {
 	}
 }
 
+Dropdown.propTypes = {
+	name: PropTypes.string.isRequired,
+	label: PropTypes.string,
+	options: PropTypes.arrayOf(PropTypes.shape({
+		name: PropTypes.string,
+		value: PropTypes.string,
+	})),
+	onChange: PropTypes.func,
```

Comment:
We call `onChange` without checking for it first, so it would be `isRequired` as well.